packages/bpk-component-skeleton/src/utils.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { CUSTOM_SIZE_TYPE } from "./common-types"
+
+const getCustomStyles = ({ height, width } : CUSTOM_SIZE_TYPE) => {
```

Comment:
I wonder if we can maybe avoid having to do this function by maybe making it clear in the API that this should be a size string of what you want. `6rem`, `4px`, etc that way it moves away from needing to have a custom function and this prop being used differently across the product so keeps it consistant for all

Reply:
Yes, only keeps strings is better.